### ISSUE TITLE
Sync doesn't work with nextcloud

### ISSUE THREAD

**Issue by adocampo** [PhieF/CarnetAndroid#29]
Now list feature is working on 0.12.0, I was trying to use Carnet as my main note-taking application.

I installed CarnetNextcloud and Carnet for android, then, I created a test note on Nextcloud and configured Carnet on Android to connect to my Nextcloud instance. So far, no problem or error, but I cannot see the note on Carnet on Android.

I tapped on "Sync with cloud storage" on Android app, but it does not seems to sync nor returns any error. Don't know if I'm doing something wrong or that message in Nextcloud logs have some meaning for you

> 
> [PHP] Error: fclose(): supplied resource is not a valid stream resource at /var/www/html/custom_apps/carnet/lib/Controller/NoteController.php#416
> 
> POST /index.php/apps/carnet/notes/metadata
> from 172.28.0.1 by malevolent at 2019-01-05T18:16:18+00:00

PS: Pulling down to refresh doesn't do anything.

**Comment by PhieF** [comment]
A fix is coming, I have to get my pull requests accepted to the nextcloud android app.

**Comment by PhieF** [comment]
Hi,

Nextcloud just released a new app on the play store, can you retry sync with carnet ?

**Comment by adocampo** [comment]
For "just" you meant last February 10th? The version I see on Google Play Store is 0.1-20190210.2226, and it doen't sync. Perhaps I need to wait until GPS updates my country...

**Comment by PhieF** [comment]
OK I mean the nextcloud official android app, not carnet

**Comment by adocampo** [comment]
Well, it seems to work...

I have both carnet and nextcloud app's latest version. I saw carnet android app was trying to sync against `Documents\QuickNote`, but my carnet's nextcloud app is pointing to `Notes`, so if I try to configure Carnet android app when the app is configured automatically, the app crashes when I try to change the sync folder, if I configure carnet android app manually, then it let me change the sync folder to match the carnet's folder on nextcloud, and then, it synced just the first note I had on nextcloud's carnet.

I decided to create a brand new directory and point both carnet apps (android and nextcloud) there. Now it seems to sync, but the syncing is quite asyncronous... I need to tap several times on the sync icon on the android app and on the nextcloud side, I need to exit and enter on the carnet app several times until the new/modified notes are shown.

So, I think we can close this issue, as carnet is in fact syncing. We should open a pair of issues more (the crashing of the android app when its self-configured and the... speed of the syncing), but having all the carnet instances pointing to the same folder, it syncs :)
